css/global.css

```diff
@@ -24,6 +24,10 @@ body {
   background-color: var(--color__background);
 }
 
+.body__dark {
+  background-color: var(--grey);
+}
+
```

Comment:
According to BEM - shouldn't you name this class `.body--dark`?

Reply:
absolutely right!